src/urllib3/util/response.py

```diff
@@ -98,4 +95,5 @@ def is_response_to_head(response):
         used 'HEAD' as a method.
     """
     # FIXME: Can we do this somehow without accessing private httplib _method?
-    return response._method.upper() == "HEAD"
+    method_str = response._method  # type: str  # type: ignore
```

Comment:
Ignore that the stubs don't know about this attribute and also cast it as a string.